recipes/aruco/3.x.x/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+set(CMAKE_CXX_STANDARD 11)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(aruco REQUIRED)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} aruco::aruco)
```

Comment:
Does aruco itself create a `Findaruca.cmake` file or `aruca-config.cmake` file?

Reply:
Yes, it provides an `arucoConfig.cmake` that provides a `aruco::aruco` target.